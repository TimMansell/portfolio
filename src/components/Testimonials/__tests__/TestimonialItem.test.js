import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import TestimonialsItem from '../TestimonialItem';

describe('TestimonialsItem', () => {
  it('should render my component', () => {
    const props = {
      description: '',
      author: ''
    };

    // eslint-disable-next-line
    const wrapper = shallow(<TestimonialsItem {...props} />);
  });

  it('should match snapshot', () => {
    const props = {
      description: 'Description',
      author: 'Author'
    };

    const snapshot = renderer.create(<TestimonialsItem {...props} />).toJSON();

    expect(snapshot).toMatchSnapshot();
  });
});
