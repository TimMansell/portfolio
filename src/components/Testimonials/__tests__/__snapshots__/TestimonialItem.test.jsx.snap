// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TestimonialsItem > should match snapshot 1`] = `
[
  <p
    className="_text_a006b6"
  >
    Description
  </p>,
  <p
    className="_name_a006b6"
  >
    Author
     (
    Position
    )
  </p>,
  <p
    className="_company_a006b6"
  >
    Company
  </p>,
]
`;
